### DISCUSSION TITLE
Warsztaty z Azure Community 2021 🚀

### DISCUSSION THREAD

**Discussion by aryll** [AzureCommunityPL/group#342] [Wydarzenia]
📢 Zapraszamy Was na warsztaty w październiku i listopadzie! Dołącz do jednej z 7 ścieżek warsztatowych i na 4 spotkaniach rozwiń swoje skrzydła w Azure!

Więcej szczegółów na https://azurecommunity.pl/warsztaty-2021/!

![Kopia Kopia Three-Frame Tea Set Photo Facebook Cover (1)](https://user-images.githubusercontent.com/5950108/134004927-05a739bc-f7b9-43f0-9bd1-7ef5a2416fd8.jpg)

**Comment by phaate** [comment]
Świetna inicjatywa! 👍
